src/map/script.c

```diff
@@ -11591,14 +11591,16 @@ BUILDIN(getstatus)
 		case 3: script_pushint(st, sd->sc.data[id]->val3); break;
 		case 4: script_pushint(st, sd->sc.data[id]->val4); break;
 		case 5:
-		{
-			const struct TimerData *td = timer->get(sd->sc.data[id]->timer);
+			if (sd->sc.data[id].infinite_duration) {
```

Comment:
I guess I need to check if some of the status timer expire or I can add else directly.
But it's best to check infinite_duration variable, since it's made for that.